Applications/PlazaDetails/Strength.qmd

```diff
@@ -9,10 +9,10 @@ The following mentioned are among the popular services available in openEO algor
 The purpose of this document is to solely provide users with an assumption on how these credits work and their strength for varying services.
 
 ##### NDVI
-For the calculation of Normalized Difference Vegetation Index (NDVI), you can acquire ~250 NDVI images, each with an area of 1 hectare, i.e. 1 NDVI image patch of 1 hectare cost 3-5 credits.
+To calculate the Normalized Difference Vegetation Index (NDVI), approximately 250 NDVI images can be acquired, each covering an area of 1 hectare. Thus, one NDVI image patch of 1-hectare costs 3-5 credits.
```

Comment:
Can simply be replaced by:

Calculating the NDVI of square hectare, costs 3-5 credits.